tests/qprogram/test_qblox_compiler.py

```diff
@@ -354,6 +362,19 @@ def test_play_named_operation_and_bus_mapping(self, play_named_operation: QProgr
         assert len(output.sequences) == 1
         assert "drive_q0" in output.sequences
         assert isinstance(output.sequences["drive_q0"], QPy.Sequence)
+    
+    def test_block_handlers(self, calibration: Calibration):
+        compiler = QbloxCompiler()
+        measurement_block = calibration.get_block("measurement")
+        measure_program = QProgram()
+        measure_program.add_block(measurement_block)
+        
+        with (
+            patch.object(QbloxCompiler, "_handle_block") as handle_block,
+        ):
+            compiler.compile(measure_program)
+
+            assert handle_block.call_count == 1
```

Comment:
I don't trust this test tbh, because it just checks that the handler is called, not that correct Q1ASM is produced. Can you add a test for the Q1ASM? Check the process with the rest of the tests for the other handlers. Essentially you can take another qprogram fixture, wrap some parts with `with qp.block():` and check that the Q1ASM does not change. Or assert the Q1ASM for this QProgram directly.